ndcube/mixins/sequence_plotting.py

```diff
@@ -736,8 +737,12 @@ def __init__(self, seq, wcs=None, axes=None, plot_axis_indices=None,
         # Add dummy axis to WCS object to represent sequence axis.
         new_wcs = utils.wcs.append_sequence_axis_to_wcs(wcs)
 
+        # Convert all the pixel values in axis_ranges to pixel_edges
+        axis_ranges_new = list()
+        for item in axes_coordinates:
+            axis_ranges_new.append(_get_extra_coord_edges(item, axis=plot_axis_indices))
```

Comment:
Looks like these three lines can be reduced into a list comprehension?

Reply:
Yeah, will do that